src/shared/messages/tracking/common.ts

```diff
@@ -3,4 +3,10 @@ export interface RawMessageData {
     date: number;
     text: string;
     html: string;
+}
+
+export interface RawMessageDataV11 {
+    id: number;
+    date: number;
+    attributes: { [key: string]: string };

```

Comment:
just fyi, `{ [key: string]: string }` is the same as `Record<string, string>` which is used all over the code base

Reply:
Thats good to know. Typescript not being my initial language, found the first result on internet that worked :+1: 
I will delete it anyway in favor of your comment above